src/gf.c

```diff
@@ -1817,6 +1817,36 @@ static int jl_type_intersection2(jl_value_t *t1, jl_value_t *t2, jl_value_t **is
     return 1;
 }
 
+enum morespec_options {
```

Comment:
I wonder if `morespec` -> `morespecific` would be worth the extra Chars. Also, what about `morespecifc_true`/`morespecific_false`/`morespecific_unknown`?

Reply:
I don't really want to deal with the code churn to rename an existing flag now. It is only used locally anyways.